packages/tools/devtools/devtools-view/package.json

```diff
@@ -57,35 +58,43 @@
 	"devDependencies": {
 		"@fluidframework/build-common": "^1.2.0",
 		"@fluidframework/build-tools": "^0.20.0",
-		"@fluidframework/build-tools": "^0.20.0",
 		"@fluidframework/core-interfaces": "workspace:~",
 		"@fluidframework/datastore-definitions": "workspace:~",
 		"@fluidframework/eslint-config-fluid": "^2.0.0",
 		"@fluidframework/shared-object-base": "workspace:~",
 		"@fluidframework/test-runtime-utils": "workspace:~",
 		"@microsoft/api-extractor": "^7.34.4",
+		"@previewjs/api": "^13.0.0",
+		"@previewjs/chromeless": "^7.0.3",
+		"@previewjs/plugin-react": "^11.0.0",
 		"@testing-library/dom": "^8.2.0",
 		"@testing-library/jest-dom": "^5.16.5",
 		"@testing-library/react": "^12.0.0",
 		"@testing-library/user-event": "^14.4.3",
 		"@types/jest": "22.2.3",
+		"@types/node": "^14.18.38",
 		"@types/react": "^17.0.44",
 		"@types/react-dom": "^17.0.18",
 		"@types/recharts": "^1.8.24",
 		"@types/testing-library__jest-dom": "^5.14.5",
+		"chalk": "^2.4.2",
 		"concurrently": "^7.6.0",
 		"copyfiles": "^2.4.1",
 		"eslint": "~8.6.0",
 		"eslint-config-prettier": "~8.5.0",
 		"eslint-plugin-jest": "~27.1.5",
 		"eslint-plugin-react": "~7.28.0",
 		"eslint-plugin-react-hooks": "~4.6.0",
+		"globby": "^13.2.2",
 		"jest": "^26.6.3",
 		"jest-junit": "^10.0.0",
+		"playwright": "^1.36.0",
 		"prettier": "~2.6.2",
+		"prop-types": "^15.8.1",
 		"rimraf": "^4.4.0",
 		"ts-jest": "^26.4.4",
-		"typescript": "~4.5.5"
+		"typescript": "~4.5.5",
+		"vite": "^4.4.3"
```

Comment:
Verify this is still needed. It was a required peer of at least some of the `previewjs` packages, but may no longer be needed.